### ISSUE TITLE
Better Multiuser support?

### ISSUE THREAD

**Issue by aslmx** [gotify/android#45]
Hey!

First of all thanks for this awesome project!

I checked out pushjet but they make people buy their App (i would, but i dont have Google in my phone 🙄)... So gotify comes in handy and i really like it.

I made an application for my smart home that can send me notifications for smoke or water detectors. 
I'd like to have my wife also receive those.
The only possible way right now seems to use one user /account on both devices. 

However i'd like to not bother my wife with my Servers status....

I suggest
-  to make it possible to share applications/tokens among multiple users
- to make notifications possible that each user can / has to Mark read / seen for him/herself

Or is it possible already somehow? 

Maybe that is something that can be combined with the multiserver feature someone asked for.

Thanks!

**Comment by jmattheis** [comment]
Hey @aslmx,

Thanks for you feedback. 

Currently it is not planned to share applications/tokes however it should be possible to implement such functionallity as plugin(gotify/server#51) (Plugin support is still in review and will still take some time to be shipped in a release see gotify/server#110)

The "mark as read" thingy already has an issue see gotify/server#21.

The easiest solution would be if you could send two messages via your application from your smart home. One to you and one to your wife.

**Comment by eternal-flame-AD** [comment]
I think this might be helpful after the plugin functionality is released. Actually one of my major reasons to implementing the plugin functionality is to provide broadcast features. You can set up a receiver filter for your wife so server broadcasts are not sent there or your wife could set up a sender filter to filter server broadcasts.

https://github.com/eternal-flame-AD/gotify-broadcast

**Comment by aslmx** [comment]
> https://github.com/eternal-flame-AD/gotify-broadcast

That looks very feature-ful/mighty but also quite complicated for such a simple use case :( 

I'll keep an eye on this. for now i'm just happy to have push notifications ;)


thanks!

**Comment by Mazvy** [comment]
+1 for this, even though it's been over a year and I see it hasn't been implemented.

I'm in the same boat as OP - I want to create multiple "apps" to differentiate notification sources, but I also want family members to receive notifications from only a few of them.  Recreating the "apps" for every user seems like a bizarre workaround - it requires to manage both Gotify and the backend for whatever is sending those notifications - copy pasting the same stuff over and over. 

Gotify is the closest I've found to what I'm want to use, but I guess Telegram + channels is the way for me.

**Comment by Mte90** [comment]
I agree with @Mazvy, I think that Gotify would be a killer project with that support.
Also with the various repository like contrib, server, android is not clear to define what is the right one to propose this feature (I was thinking to put little money for a bounty).
